pkg/e2e/operators/splunkforwarder.go

```diff
@@ -163,7 +164,12 @@ func dedicatedAaddSplunkforwarder(SplunkForwarder sfv1alpha1.SplunkForwarder, na
 		return err
 	}
 	unstructuredObj := unstructured.Unstructured{obj}
-	h.SetServiceAccount("system:serviceaccount:%s:dedicated-admin-project")
+	h.Impersonate(rest.ImpersonationConfig{
+		UserName: "test-user@redhat.com",
+		Groups: []string{
+			"dedicated-admins",
+		},
+	})
```

Comment:
I can't see anywhere where it's resetting the impersonation when the test finishes. There should likely be something like the following, in keeping with the convention that other tests do.

```
      defer func() {
        h.Impersonate(rest.ImpersonationConfig{})                                                                                                                                                               
      }()
```

Reply:
Added the reset step. Thanks for reviewing, @mrbarge !